src/commands/generate/generateCommand.ts

```diff
@@ -146,7 +146,7 @@ export class GenerateCommand implements CommandModule {
             throw Error('Must either specify "rawSource" or specify "packageName" and "packageVersion"');
         } else if (!fs.existsSync(output)) {
             throw new Error(`Directory "${output}" does not exist`);
-        } else if (!rawSource && !isValid(packageVersion!)) {
+        } else if (!rawSource && !SlsVersion.isValid(packageVersion!)) {
```

Comment:
Broke my own no-code-change threshold, but this seemed reasonable & simple for a palantir dep